Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1523,7 +1523,7 @@ public static partial class PythonOps {
                 throw PythonOps.CreateThrowable(DynamicHelpers.GetPythonType(obj));
             }
 
-            throw PythonOps.AssertionError();
+            throw PythonOps.AssertionError("");
```

Comment:
I don't think raising an `AssertionError` with a string is technically correct, but this code path is pretty much never hit (unless you delete `__builtins__.AssertionError`), so I'm fine with this for now. I'll file an issue with the problems I found with our assert implementation.